README.md

```diff
@@ -4,7 +4,27 @@ Exercism problems in C
 
 ## Contributing Guide
 
-Please see the [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data)
+Please see the Exercism [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data) for general contribution tips.
```

Comment:
I think this should point to the x-common contributing guide, https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md

That seems to be where the most recent instructions are.